### PR TITLE
fix: fix network endpoints

### PR DESCRIPTION
**Pull request by Hemanthghs**

### CODE REVIEW

File: frontend/src/pages/overview/overview/ChainsOverview.js [hunk 1]
```diff
@@ -181,7 +181,7 @@ export const ChainsOverview = ({ chainNames }) => {
         const denomInfo = chainDenoms[chainName]?.filter((denomInfo) => {
           return denomInfo.denom === balance.denom;
         });
-        if (balance.denom !== minimalDenom && denomInfo.length) {
```
Comment (Hemanthghs) [inline frontend/src/pages/overview/overview/ChainsOverview.js:184]: Page got broken when denom info was null

File: frontend/chains/agoric.json [hunk 2]
```diff
@@ -21,8 +21,8 @@
     "config": {
       "chain_id": "agoric-3",
       "chain_name": "Agoric",
-      "rest": "https://api.resolute.vitwit.com/agoric_api/",
-      "rpc": "https://api.resolute.vitwit.com/agoric_rpc/",
+      "rest": "https://agoric-api.polkachu.com",
```
Comment (aleem1314) [inline frontend/chains/agoric.json:24]: Why are we using another provider? Is our endpoint not working?

Comment (Hemanthghs) [reply frontend/chains/agoric.json:24]: Yes our endpoint is returning connection refused error

File: frontend/chains/akash.json [hunk 3]
```diff
@@ -21,8 +21,8 @@
   "config": {
     "chain_id": "akashnet-2",
     "chain_name": "Akash",
-    "rest": "https://api.resolute.vitwit.com/akash_api/",
-    "rpc": "https://api.resolute.vitwit.com/akash_rpc/",
+    "rest": "https://api.resolute.vitwit.com/akash_api",
```
Comment (aleem1314) [inline frontend/chains/akash.json:24]: This should be handled at the application level by performing validation. If there is `/` at the end, we should remove it.

**Review by aleem1314 — APPROVED** [review]

File: frontend/chains/akash.json (hunk 3 above)
Comment (Hemanthghs) [reply frontend/chains/akash.json:24]: Okay aleem

**Review by aleem1314 — APPROVED** [review]

**Review by aleem1314 — APPROVED** [review]

**Pull request merged**